### PR TITLE
Update repo using new version of generator

### PR DESCRIPTION
**Pull request by jordanjones243**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Fixes:** #47, #103, #110 

## Summary:

_Please summarize the scope of the changes you have submitted, what the intent of the work is and anything that describes the before/after state of the project._

* Update repo to support Node 18+ and other modernized standards using the new version of the `WC-Generator`.
* Add csspart to the `fieldset` element of `auro-radio-group`.
* Make disabled `auro-radio` inputs non focusable

## Type of change:

Please delete options that are not relevant.

- [x] New capability
- [x] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

**Review by blackfalcon — CHANGES_REQUESTED** [review]
I added a commit that updates the main demo page. When I ran the api.html file, there are a lot of private properties and methods being exposed. These need to be updated in the code. 

Also, the api.html page is not fully flushed out. Most of the links do not go anywhere.

**Comment by blackfalcon** [comment]
Please also see this update https://github.com/AlaskaAirlines/auro-radio/pull/109/files and add to this PR.

File: docs/partials/demo.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+import './node_modules/@alaskaairux/auro-radio';
+registerComponent('custom-radio');
+```
+
+This will create a new custom element that you can use in your HTML that will function identically to the `<auro-radio>` element.
+
+Using the `registerComponent` function to create a custom `<auro-radio>` will also create a custom `<auro-radio-group>` with the exact same naming convention, ending in "-group". For example, using `registerComponent('custom-radio')` will result in `<custom-radio-group>` also being created.
+
+<div class="exampleWrapper">
+  <!-- AURO-GENERATED-CONTENT:START (FILE:src=./../../apiExamples/customRadio.html) -->
```
Comment (jordanjones243) [inline docs/partials/demo.md:177]: The custom version of auro-radio is not behaving correctly, circle back around to this to figure out the issue

**Review by jason-capsule42 — CHANGES_REQUESTED** [review]
New file requested by Dale `.npmrc` in an earlier comment was not addressed.

The the `demoPublish` workflow needs to be added.

**Comment by jordanjones243** [comment]
> New file requested by Dale `.npmrc` in an earlier comment was not addressed.
> 
> The the `demoPublish` workflow needs to be added.

`.npmrc` has already been added in a previous commit, `publishDemo.yml` now added.

**Review by jason-capsule42 — APPROVED** [review]

**Pull request merged**